frontend/src/utils/common.ts

```diff
@@ -133,3 +136,52 @@ export const currentChainSupportsQuests = () => {
     getConfigValue('VUE_APP_SIMPLE_QUESTS_CONTRACT_ADDRESS') || (simpleQuestsNetworks as Networks)[networkId]?.address;
   return !!contractAddress;
 };
+
+export const currentChainSupportsPartnerVault = () => {
+  const networkId = getConfigValue('VUE_APP_NETWORK_ID') || '5777';
+  const contractAddress = process.env.VUE_APP_PARTNER_VAULT_CONTRACT_ADDRESS ||
+    getConfigValue('VUE_APP_PARTNER_VAULT_CONTRACT_ADDRESS') || (partnerVaultNetworks as Networks)[networkId]?.address;
+  return !!contractAddress;
+};
+
+export const getTimeRemaining = (end: string) => {
```

Comment:
This looks like a duplicate of already existing function `secondsToDDHHMMSS`, you'd just need to pass `end - Date.now()/1000` instead of `end`.

Reply:
I'm using different formatting later on in multiple places, this function allows me to get all the numerical values I need